packages/web/src/shared/components/form/ComboboxInput.tsx

```diff
@@ -6,97 +6,36 @@ import {
   useEffect,
   useState,
 } from 'react';
-import { Controller, useFormContext } from 'react-hook-form';
+import { useFormContext } from 'react-hook-form';
 import { useTranslation } from 'react-i18next';
 import { Icon } from '../Icon';
 
-const CREATE_TAG = '_create';
 const MAX_ITEMS = 1000;
 
 export interface ComboboxItem {
   label: string;
   value: string;
 }
 
-export type ComboboxInputProps = BaseComboboxInputProps & {
-  required?: boolean;
-  isolate?: boolean;
-};
-
-export default function ComboboxInput(props: ComboboxInputProps) {
-  const context = useFormContext();
-
-  if (context && !props.isolate) {
-    return (
-      <Controller
-        control={context.control}
```

Comment:
We also don't need this wrapper component that decides whether or not to use the Controller or not. That is what got us into trouble with the sortable list input because the presence of the form context caused the input to register itself when we didn't want it to